docs/user-guide/connectors/mattermost.rst

```diff
@@ -50,8 +58,7 @@ you need to supply a ``credentials.yml`` with the following content:
    mattermost:
      url: "https://chat.example.com/api/v4"
      team: "community"
```

Comment:
@btotharye do we still need to define the team? since the bot is an integration already bind to the team, it may not be necessary. I was checking the API here https://api.mattermost.com/#tag/bots don't know if there is something I do not see.

Reply:
maybe not, for now we can leave it in probably until I test it further.  I'd have to test some of the calls and see but I thought it might have been needed for some of the calls, maybe not anymore now.

Reply:
I thought with how the mattermost API worked it will need this team I can't remember without going to look into it further.

Reply:
I double checked right now my wrapper is still expecting something for this so I'll have to eventually update that so for now we can probably just leave this in and I'll work on fixing that in a later fix since I would have to update my wrapper again.